SpatialGDK/Documentation/content/game-client-interest-management.md

```diff
@@ -0,0 +1,111 @@
+<%(TOC)%>
+# Game client interest management
+
+The GDK uses [interest]({{urlRoot}}/content/glossary#interest) to control the information that [game clients]({{urlRoot}}/content/glossary#client-workers) receive about the world around them.
+
+Game clients need information about Actors that they don’t own (see the Unreal documentation on [owning](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/OwningConnections) to help them to correctly manipulate the Actors that they _do_ own, and to render the SpatialOS world. Interest enables game clients to receive the relevant information.
```

Comment:
```suggestion
Game clients need information about Actors that they don’t own (see the Unreal documentation on [owning](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/OwningConnections)) to help them to correctly manipulate the Actors that they _do_ own, and to render the SpatialOS world. Interest enables game clients to receive relevant information from Actors.
```
Missing parenthesis, a slight change to the last sentence.

Reply:
Thanks! Actually I might update to:

> receive the relevant information _about these Actors_

rather than "from Actors", because when we talk about receiving info _from_ somewhere/something, we tend to mean the entity database. (I guess it's a direct vs indirect "from")